src/patternfly/components/Alert/alert.scss

```diff
@@ -34,9 +34,9 @@
   --#{$alert}__icon--FontSize: var(--pf-t--global--icon--size--md);
 
   // Title
-  --#{$alert}__title--FontFamily: var(--pf-t--global--font--family--heading);
-  --#{$alert}__title--FontSize: var(--pf-t--global--font--size--heading--xs);
-  --#{$alert}__title--FontWeight: var(--pf-t--global--font--weight--heading);
+  --#{$alert}__title--FontFamily: var(--pf-t--global--font--family--body);
+  --#{$alert}__title--FontSize: var(--pf-t--global--font--size--body--default);
+  --#{$alert}__title--FontWeight: var(--pf-t--global--font--weight--body--bold);
   --#{$alert}__title--LineHeight: var(--pf-t--global--font--line-height--heading);
```

Comment:
We should remove this `line-height--heading` var if we don't expect users to theme the font-family. It was only there before because we always declare line-height use `line-height--heading` any time we use the `font--family--heading` token. PF global styles set `line-height--body` on everything by default.